assets/constants.ts

```diff
@@ -5237,6 +5246,32 @@ export default {
     ]
   }],
 
+  // These are linked to the VALID_INTERACTION_IDS constant in
```

Comment:
To keep these & the others linked, can we have tests to verify that they generally match? It seems important to ensure that the two are kept in sync, and part of having the validation file in place is to make sure that Android developers become aware of when such changes are needed.

Reply:
Yup, added tests to `android_validation_constants_test`.